spec/controllers/oauth/applications_controller_spec.rb

```diff
@@ -6,6 +6,7 @@ module Oauth
     let!(:admin) { FactoryBot.create :user, :terms_agreed, :admin }
     let!(:user)  { FactoryBot.create :user, :terms_agreed }
     let!(:user2) { FactoryBot.create :user }
+    let!(:user3) { FactoryBot.create :user, :terms_agreed }
```

Comment:
I see that you are following the user variable naming pattern, but here it might be nice to give this user a meaningful name (for whatever its purpose is in this spec)

Reply:
renamed